packages/template-retail-react-app/app/components/swatch-group/index.jsx

```diff
@@ -28,9 +35,7 @@ const SwatchGroup = (props) => {
                     const childValue = child.props.value
 
                     return React.cloneElement(child, {
-                        selected: childValue === value,
-                        key: childValue,
-                        value,
```

Comment:
Aside: there was a subtle bug with the original code. Now the parent (SwatchGroup) no longer overriding the children's `value`. They should be distinct.